examples/seismic/tutorials/11_viscoacoustic.ipynb

```diff
@@ -246,8 +246,8 @@
    "outputs": [],
```

Comment:
<p>Is this rendering for you?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
yes